src/redux/reducers/token.ts

```diff
@@ -36,6 +37,7 @@ export default function (state: State = initialState, action) {
         tokens = Token.replaceToken(state.tokens, token);
       } else {
         tokens = Token.addAccountToTokens(state.tokens, account);
+        LocalStorage.setTokens(tokens);
```

Comment:
This should be outside of if else.
Updated tokens should be updated in LocalStorage